package.json

```diff
@@ -3,7 +3,7 @@
     "lerna": "3.0.0-beta.21"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap --hoist tap --hoist eslin* --hoist husky --hoist prettier --hoist pretty-quick",
+    "bootstrap": "lerna bootstrap --no-ci --hoist tap --hoist eslin* --hoist husky --hoist prettier --hoist pretty-quick",
```

Comment:
What does this do? I cannot find this flag on https://github.com/lerna/lerna

Reply:
Shouldn’t we move these to the lerna.json config in the root so these flags always get added?

Reply:
The `--no-ci` flag is part of v3, those docs are v2. https://github.com/lerna/lerna/tree/v3.0.0-beta.21#--ci

When I bumped npm from 5 to 6, lerna uses `npm ci` instead of `npm install`, and I had error messages surrounding it. Since I removed the Travis npm update, I also removed this tweak.